<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NV08C RTK config 1.0.0.2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="rowCount">
       <number>28</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>301</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <item row="0" column="0">
       <property name="text">
        <string>Description</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>226</red>
          <green>226</green>
          <blue>226</blue>
         </color>
        </brush>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>Value</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>226</red>
          <green>226</green>
          <blue>226</blue>
         </color>
        </brush>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>PNVGVER: RTK FW version, CSM FW version</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>PNVGNME: Number of digits in fractional part of time output, permitted values 0 to 6</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>PNVGNME: Number of digits in fractional part of position output, permitted values 0 to 9</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>PNVGNME: Set compatibility mode for Talker ID:
0 – Talker ID for GGA and RMC always “GP” (by default)
1 – Compatibility mode Off. Talker ID as specified in messages description Note – The field is optional and can be omitted
</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>PNVGRZA,1 UART1: Port baud rate, in bauds from 4,800 to 460,800</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>PNVGRZA,1 UART1: Protocol type</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string>PNVGRZA,2 UART2: Port baud rate, in bauds from 4,800 to 460,800</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="7" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="8" column="0">
       <property name="text">
        <string>PNVGRZA,2 UART2: Protocol type</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="8" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="9" column="0">
       <property name="text">
        <string>PNVGRZA,3 USB: Port baud rate, in bauds from 4,800 to 460,800</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="9" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="10" column="0">
       <property name="text">
        <string>PNVGRZA,3 USB: Protocol type</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="10" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="11" column="0">
       <property name="text">
        <string>PNVGRTK,MODE: 
Setting of operation mode (0 – Autonomous Mode, 1 – Base Station Mode, 2 – RTK-Rover Mode (by default), 3 – Base Station Mode with automatic antenna position averaging)
</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="11" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="12" column="0">
       <property name="text">
        <string>PNVGRTK,PVTRATE: PVT rate, Hz (1, 2, 5, 10)</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="12" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="13" column="0">
       <property name="text">
        <string>PNVGRTK,SYSGPS: 1 – GPS ON (by default), 0 – GPS OFF</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="13" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="14" column="0">
       <property name="text">
        <string>PNVGRTK,SYSGLO: 1 – GLONASS ON (by default), 0 – GLONASS OFF</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="14" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="15" column="0">
       <property name="text">
        <string>PNVGRTK,ELGPS: GPS Elevation Mask, degrees (5 by default)</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="15" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="16" column="0">
       <property name="text">
        <string>PNVGRTK,ELGLO: GLONASS Elevation Mask, degrees (5 by default)</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="16" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="17" column="0">
       <property name="text">
        <string>PNVGRTK,SNR: SNR Mask, dB-Hz (33 by default)</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="17" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="18" column="0">
       <property name="text">
        <string>PNVGRTK,BASEID: Station ID for Base mode, 0 - 4095 (2 by default)</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="18" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="19" column="0">
       <property name="text">
        <string>PNVGRTK,BASEXYZ: Base Station Antenna Reference Point ECEF-X,Y,Z, m </string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="19" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="20" column="0">
       <property name="text">
        <string>PNVGRTK,BASEBLH: Base Station Antenna Reference Point. Two formats are supported:
  B,L,H (radians for B and L, metres for H)
  NMEA message format
</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="20" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="21" column="0">
       <property name="text">
        <string>PNVGRTK,AVGTIME: Antenna position averaging time for Base mode, minutes</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="21" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="22" column="0">
       <property name="text">
        <string>PNVGRTK,AVGMODE: Type of coordinates used for antenna position averaging:
0  – any valid coordinates, 1 – Fixed &amp; Float, 2 – Fixed only
</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="22" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="23" column="0">
       <property name="text">
        <string>PNVGRTK,SFMODE: Secondary filter mode:
0 – DISABLE, 1 – RETAIN, 2 – TRANSITION, 3 – HYBRID
</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="23" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="24" column="0">
       <property name="text">
        <string>PNVGRTK,SFTRS: Transfer Speed for Secondary Filter, m/s</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="24" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="25" column="0">
       <property name="text">
        <string>PNVGRTK,ICBGLO: GLONASS Inter-Channel Bias. Value is defined as shift between phases of nearest GLONASS carriers in parts of phase cycle</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="25" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="26" column="0">
       <property name="text">
        <string>PNVGRTK,ICBMODE: GLONASS Inter-Channel Bias calibration mode:
0 – OFF
1 – initial calibration (one time calibration using GPS as reference)
2 – real-time recalibration (in parallel with GPS&amp;GLONASS Fix)
</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="26" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="27" column="0">
       <property name="text">
        <string>PNVGRTK,ICBTIME: GLONASS Inter-Channel Bias calibration interval (length of one step of calibration), seconds</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="27" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="Serial_comboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Baud_comboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Connect_pushButton">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Request_pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Request</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Set_pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Set </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Default_pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Default Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="VRS_pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>VRS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Save_pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save to Flash</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveToFile_pushButton">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string>Save to File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LoadFromFile_pushButton">
        <property name="text">
         <string>Load from File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
